### PR TITLE
Added license.gradle and maven.gradle

### PR DESCRIPTION
**Pull request by modmuss50**
Just here to be checked over

### CODE REVIEW

File: gradle/license.gradle [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+		classpath "gradle.plugin.net.minecrell:licenser:0.2.1"
+    }
+}
+
+//Due to a gradle bug, the full class name needs to be used
+apply plugin: net.minecrell.gradle.licenser.Licenser
```
Comment (shadowfacts) [inline gradle/license.gradle:13]: Use

``` groovy
plugins {
   id "net.minecrell.licenser" version "0.2.1"
}
```

for simplicity

File: gradle/license.gradle [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+license {
+    header file('HEADER')
+    include '**/*.java'
+}
```
Comment (jamierocks) [inline gradle/license.gradle:18]: newline

File: gradle/maven.gradle [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                            id = 'modmuss50'
+                            name = 'modmuss50'
+                            roles { role 'developer' }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```
Comment (jamierocks) [inline gradle/maven.gradle:54]: newline

File: gradle/license.gradle (hunk 1 above)
Comment (modmuss50) [inline gradle/license.gradle:13]: Doesn't work with sub scripts

Comment (jamierocks) [inline gradle/license.gradle:13]: No but this isn't either, just an example...

**Pull request merged**